deployment_report/templates/viya_deployment_report.html.j2

```diff
@@ -299,14 +308,22 @@
   <h2>Configurations</h2>
   {# Configurations: Content #}
 
-  {# Cluster Overview: Configmaps Accordion #}
-  {% set kind_name = "Configmaps" %}
   <div class="jq-accordion">
-    <h2>Configmaps</h2>
-    <div>
-      {% for name, info in report_data.kubernetes.configMaps.items() %}
+  {# Configurations: Configmaps Accordion #}
+  <h2>Configmaps</h2>
+  <div>
+  {% set kind_name = "Configmaps" %}
+  {% if report_data.kubernetes.configMaps|length == 0 %}
+    <div class="notification-box info-box">
+      <i class="fa fa-info-circle notification-box-icon" aria-hidden="true"></i>
+      <div class="notification-box-text margin-block-end-1em">
+        No configMaps were found in namespace [{{ report_data.kubernetes.namespace }}].
```

Comment:
Let's change `configMaps` to `ConfigMaps` to better align with the way k8s formats the name.